apps/client/src/components/VisualStyles.js

```diff
@@ -7,6 +8,26 @@ export const FigCaption = styled.figcaption`
 export const Figure = styled.figure`
   min-height: 1px; /* IE11 Bug */
   margin: 0 0 24px 0;
+  ${({ isLoading, theme, animateLoading, backgroundImage }) =>
```

Comment:
Shameless copy/paste from `ImageCardStyle` from asc-ui :-P

Why keep `theme, animateLoading, backgroundImage` if it's never defined and passed to this function?